### DISCUSSION TITLE
[Query]: Merge two tenants DB

### DISCUSSION THREAD

**Discussion by felixfenu** [snipe/snipe-it#12622] [Q&A]
### Is your feature request related to a problem? Please describe.

Hi Everyone!
I am in the process of merging two Snipe-IT databases (two different tenants into one) and I would like to know what are the best practices to do it.
What is the recommended process? Especially thinking about FK in database
Eg. Manufacturer id 1 in tenant 1 is Dell, but
Manufacturer id 1 in tenant 2 is Lenovo

And so also with the other tables

### Describe the solution you'd like

Hi Everyone!
I am in the process of joining two Snipe-IT databases (two different tenants into one) and I would like to know what are the best practices to do it.
What is the recommended process? Especially thinking about FK in database
Eg. Manufacturer id 1 in tenant 1 is Dell, but
Manufacturer id 1 in tenant 2 is Lenovo

And so also with the other tables

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
